### PR TITLE
Add support for Ubuntu 18

### PR DESCRIPTION
**Pull request by jcsiadal**
This appears to be a regression error. New versions of Perl5 require left braces to be escaped.

This was fixed in 5cadafa38580a8bb4b28f5c602e0566c07f15f2a on Aug 13, 2018. Merged in 1ed8f1a4c4c1d9c14ba1fd8fef3a9c2424e0ba66 on Sept 10, 2018.

Appear to have regressed in 2d5796333599c66012f90c1f66f04b5f0e036f2f on Apr 19, 2019. Only Isc.pm was impacted in this commit; the other two files in the original fix were not changed.

### CODE REVIEW

**Review by bensallen — DISMISSED** [review]
Thanks for the fix.

**Comment by jcsiadal** [comment]
Sorry, I forgot to rebranch before committing the Ubuntu updates. This fix has to be in place to support Ubuntu 18. Submitting a pull request for the Ubuntu 18 update in a few minutes.

**Comment by jcsiadal** [comment]
Adds support for Ubuntu 18. Tested with Ubuntu 18.04. Added an example build-script under platform used to build and install on the head node. Most importantly, the script has the complete package list needed for successful build and installation. The automated deb installation script was not modified as it will take some effort to update.

Left braces must be escaped in Perl5 modules. The Isc.pm module is fixed.
The current ipmitool source will not compile in Ubuntu-18. Use the distribution ipmitool.

The ubuntu-18 VNFS template is added. Identical to 16.04 except for one package name.

The ubuntu common template is updated:
-Replaced use of chroot option in dpkg with direct chroot calls to apt-get. The python36-minimal package failed to install using dpkg chroot configuration. Using chroot to call apt-get produced no errors.
-Turned off interactive package configuration; otherwise, one package will cause an error.

Validated on 4-node cluster using similar manual installation steps done in CentOS 7. SELinux and firewall disabled, so not yet tested in secure configuration.

File: vnfs/libexec/wwmkchroot/include-ubuntu [hunk 1]
```diff
@@ -2,7 +2,8 @@
 APT_CONF="root/ww-apt.conf"
 DEB_SRCS="root/sources.list"
 DEB_ISOS="root/iso.list"
-APT_CMD="apt-get -c $CHROOTDIR/$APT_CONF -o Dir::Etc::SourceList=/$CHROOTDIR/"
+#APT_CMD="env DEBIAN_FRONTEND="noninteractive" apt-get -c $CHROOTDIR/$APT_CONF -o Dir::Etc::SourceList=/$CHROOTDIR/"
```
Comment (bensallen) [inline vnfs/libexec/wwmkchroot/include-ubuntu]: Can you remove the commented out lines here and below? They'll be in git's history so no need to keep them preserved in-situ.

**Review by bensallen — APPROVED** [review]
Thanks

**Pull request merged**